dev-libs/libmowgli/libmowgli-2.1.0.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Useful set of performance and usability-oriented extensions to C"
+HOMEPAGE="https://github.com/atheme/libmowgli-2"
+SRC_URI="https://github.com/atheme/libmowgli-2/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+IUSE="ssl"
+
+LICENSE="BSD-2"
+SLOT="2"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos"
+RDEPEND="ssl? ( dev-libs/openssl:* )"
```

Comment:
There has been an interface change between 2.1.0 and 2.0.0, and I'd imagine this would be taken care of by the library version numbers and not slotting, yet slot 2 is also what the 2.0.0 ebuild uses.  I'll switch it to slot "0".